lib/solidus_subscriptions/subscription_generator.rb

```diff
@@ -54,6 +56,15 @@ def group(subscription_line_items)
 
     private
 
+    def cleanup_subscription_line_items(subscription_line_items)
+      ids = subscription_line_items.pluck :id
+      SolidusSubscriptions::LineItem.where(id: ids).update_all(
```

Comment:
Why didn't you use `subscription_line_items.update_all(...)` directly? 🤔

Reply:
The problem that I see is that those line_items come from the `group` method of the same class ([ref](https://github.com/solidusio-contrib/solidus_subscriptions/blob/master/lib/solidus_subscriptions/subscription_generator.rb#L48)) which returns an array, not `ActiveRecord::Relation`.